### PR TITLE
set grpc stream_init_window_size to 2MB

### PR DESCRIPTION
**Pull request by javaforfun**

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

File: src/server/server.rs [hunk 1]
```diff
@@ -104,6 +105,7 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver> Server<T, S> {
         let addr = try!(SocketAddr::from_str(&cfg.addr));
         let ip = format!("{}", addr.ip());
         let channel_args = ChannelBuilder::new(env.clone())
+            .stream_initial_window_size(STREAM_INITIAL_WINDOW_SIZE)
```
Comment (BusyJay) [inline src/server/server.rs]: Raft client should be updated too.

Comment (siddontang) [inline src/server/server.rs]: any update, @javaforfun ?

Comment (javaforfun) [inline src/server/server.rs]: seems we need add config for raft_client

**Comment by siddontang** [comment]
Please fix the conflict and I suggest adding this to the configuration file too.

**Comment by javaforfun** [comment]
I mean put duplicate const var config to server::Config,  not defining it in many files.

**Comment by siddontang** [comment]
> not defining it in many files.

sorry, what does it mean? @javaforfun

**Pull request merged**